src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: codelab
+title: Use a Service Worker to manage notifications
+authors:
+  - katejeffreys
+description: |
+  In this codelab, learn how to manage notifications with a
+  service worker.
+date: 2019-10-14
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-service-worker
+# Specify which file the glitch should start on.
+glitchPath: public/index.js  
+related_post: use-push-notifications-to-engage-users
+draft: true
+---
+
+In this codelab, you'll use a service worker
+to manage [notifications](/use-push-notifications-to-engage-users).
+The instructions here assume that you're already familiar with
+service workers and the basics of requesting notification permission and sending notifications.
+If you need a refresher on notifications, see the
+[Get started with the Notifications API](/codelab-notifications-get-started)
+codelab. To learn more about service workers, see Matt Gaunt's [Introduction to service workers](https://developers.google.com/web/fundamentals/primers/service-workers).
+
+## Remix the sample app and view it in a new tab
+
+Notifications are automatically blocked from the embedded Glitch app, so you won't be able to preview the app on this page. Instead, here's what to do:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+
+The Glitch should open in a new Chrome tab.
+
+As you work through this codelab, make changes to the code in the embedded Glitch on this page. Refresh the new tab with your live app to see the changes.
+
+## Get familiar with the sample app and starting code
+
+Start by looking at the live app in the new Chrome tab:
```

Comment:
```suggestion
Start by looking at the live app in the new browser tab:
```

Reply:
Same comment as above: steps rely on Chrome DevTools, so users really have to use Chrome.